plugins/woocommerce/e2e/tests/merchant/order-emails.spec.js

```diff
@@ -7,7 +7,11 @@ test.describe( 'Merchant > Order Action emails received', () => {
 	const customerBilling = {
 		email: 'john.doe.merchant.test@example.com',
 	};
-	const adminEmail = 'admin@woocommercecoree2etestsuite.com';
+
+	const adminEmail =
```

Comment:
This check here is necessary to ensure that the tests work with both `e2e-environment` and `wp-env`. The reason for this is because the `wp-env` environment will always set the admin email to `wordpress@example.com` which would break the tests. 

This value should be configurable when setting up `wp-env` but that's currently now the case. I've submitted a [PR](https://github.com/WordPress/gutenberg/pull/42421) to the Gutenberg repo to fix this issue, but unfortunately, we will have to stick with this until that PR is merged and makes it way into a new version of `wp-env`.

Reply:
Makes sense.  Or we can set it in a config file centrally for the tests, and switch between configs based on the environment..

Reply:
Yea. A `config.js` file would be ideal here. I figured we could address that and all the other hard coded test data in a separate PR.